projects/uitpas/reference/uitpas.json

```diff
@@ -3651,6 +3651,15 @@
             "style": "form",
             "explode": true
           },
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "name": "sort",
+            "description": "Sorts the rewards in a specific order. Possible values are: `name`, `creationdate`, `redeemCount`, `featured`, `points`. Multiple values can be combined in a comma separated string so the rewards are first sorted by the first field, then the second and so on. By default the sort uses ascending order. Descending order can be specified by including a `-` sign before the field that needs to be ordered descending. e.g. use `-redeemCount,points` to sort descending on redeemCount, and next ascending on `points`.",
```

Comment:
Description is pretty clear but I feel like an `example` could also be useful for this parameter. See https://swagger.io/docs/specification/describing-parameters/ > "Parameter Examples"